traitcuration/settings.py

```diff
@@ -118,3 +118,6 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Activate Django-Heroku.
+django_heroku.settings(locals())
```

Comment:
PEP8 suggests that all files should add with a newline, could you please add that? Also for `Procfile`